examples/lights/lights.c

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (C) 2009 Ubixum, Inc. 
+ * Copyright (C) 2016 Matthew Iselin <miselin@google.com>
+ * Copyright (C) 2017 Kyle Robbertze <krobbertze@gmail.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ **/
+
+#include <fx2regs.h>
+#include <delay.h>
+
+void init_lights(void) {
+    IFCONFIG &= ~3;
```

Comment:
`3 == 1b00000011` in binary. `~3 == 1b11111100` -- Using `&=` means they are clearing the bottom two bits. IFCONFIG is a special register -- you should look up what these bits are used for.